src/utils/jsonTransform.test.js

```diff
@@ -0,0 +1,315 @@
+import {to_registration_service_format} from './jsonTransform';
```

Comment:
file naming conventions in this project use hyphens. We should change the name to `json-transform`.